e2e/test.js

```diff
@@ -0,0 +1,90 @@
+"use strict";
+import { Selector, ClientFunction } from 'testcafe';
+import MainPage from './main_page.js';
+import GamePage from './game_page.js';
+
```

Comment:
Not reviewing (yet) since I don't understand JavaScript tests :(

Reply:
These aren't per se the Javascript tests (those are in frontend/test/unit/), this are the End-to-end tests, which attempt to behave like a normal user and check that things "work".